docs/src/equations.md

```diff
@@ -65,13 +65,16 @@ We make use of the following operators
 * ``\bar{\boldsymbol{u}} = \boldsymbol{u}_h + I_{c}(\boldsymbol{u}_v)``
 
 * ``\Phi = g z`` is the geopotential, where ``g`` is the gravitational acceleration rate and ``z`` is altitude above the mean sea level.
+* ``\boldsymbol{b} = - (\rho - \rho_h) / \rho \nabla \Phi`` is the reduced gravitational acceleration
```

Comment:
this is a little confusing: is `nabla \Phi` in the numerator or denominator? Maybe use `\frac` and make it a ```math block

Reply:
Done.